templates/import/vanilla.template.yml

```diff
@@ -8,9 +8,9 @@ hooks:
     - exec:
         cd: /etc/service
         cmd:
-          - rm -R unicorn
-          - rm -R nginx
-          - rm -R cron
+          - echo "" > unicorn/run
+          - echo "" > nginx/run
+          - echo "" > cron/run
```

Comment:
Oh, by the way, I think this could actually be replaced with the following snippet:

```yaml
env:
  UNICORN_SIDEKIQS: 0
```

It needs to be at the top of the template and the template needs to be included at after `"templates/web.template.yml"` in the container config. For simplicity, I'd put it at the end of the templates list in the container config.